README.md

```diff
@@ -9,7 +9,7 @@ React-admin usually requires a REST/GraphQL server to provide data. In this demo
 To explore the source code, start with [src/App.tsx](https://github.com/marmelab/react-admin/blob/master/examples/crm/src/App.tsx).
 
 
-## Setup
+## Local Supabase Setup
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** ‏Should we be talking about the `config.toml` file?

Reply:
Probably, since I did not worked on the local configuration I don't know if there are too much to configure. Does-it require some setup to work or does it work out of the box ?

Reply:
Since the password reset feature, some optional config is available, e.g. to customize the email template. But it should still work out of the box though.